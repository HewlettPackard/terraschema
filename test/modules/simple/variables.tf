variable "name" {
    type        = string
    description = "Your name."
    default = "world"
}

variable "age" {
    type        = number
    description = "Your age. Required."
}