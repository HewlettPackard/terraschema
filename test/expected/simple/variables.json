{
	"age": {
		"default": null,
		"description": "Your age. Required.",
		"type": "number"
	},
	"name": {
		"default": "world",
		"description": "Your name.",
		"type": "string"
	}
}
